iocage_cli/__init__.py

```diff
@@ -179,8 +179,13 @@ def list_commands(self, ctx):
         return rv
 
     def get_command(self, ctx, name):
-        mod = __import__(f"iocage_cli.{name}", None, None, ["cli"])
-        mod_name = mod.__name__.replace("iocage_cli.", "")
+
+        try:
+            mod = __import__(f"iocage_cli.{name}", None, None, ["cli"])
+            mod_name = mod.__name__.replace("iocage_cli.", "")
+        except ImportError:
+            # No such command
```

Comment:
Perhaps we should do a SystemExit() here saying that?

Reply:
The click-module already does a nice job handling this, see output below.  

```
(iocage-env) ➜  iocage foo 
Usage: iocage [OPTIONS] COMMAND [ARGS]...
Try "iocage --help" for help.

Error: No such command "foo".
```
Are you OK with that?

Reply:
Ah! Yes :)